react-common/components/extensions/ExtensionCard.tsx

```diff
@@ -3,6 +3,7 @@ import { Button } from "../controls/Button";
 import { Card } from "../controls/Card";
 import { LazyImage } from "../controls/LazyImage";
 import { classList } from "../util";
+import { Link } from "../controls/Link";
```

Comment:
Should we remove the Button from the import since it's no longer being used here?